eZ/Publish/Core/MVC/Symfony/Security/UserWrapped.php

```diff
@@ -106,6 +127,22 @@ public function eraseCredentials()
 
     public function isEqualTo(CoreUserInterface $user)
     {
+        if ($user instanceof self) {
+            return $this->wrappedUser instanceof EquatableInterface ? $this->wrappedUser->isEqualTo($user->wrappedUser) : true;
+        }
+
         return $this->wrappedUser instanceof EquatableInterface ? $this->wrappedUser->isEqualTo($user) : true;
```

Comment:
not sure when this would ever be the case

Reply:
According to method signature it's possible. I would maybe simplify first `if` statement to 

```
        if ($user instanceof self) {            
            $user = $user->wrappedUser;
        }
```

to avoid duplication of `$this->wrappedUser instanceof EquatableInterface ?` logic.